tools/nodejs_api/CMakeLists.txt

```diff
@@ -43,3 +43,12 @@ file(COPY ${CMAKE_JS_SRC} DESTINATION ./kuzu)
 add_library(${PROJECT_NAME} SHARED ${SOURCE_FILES} ${CMAKE_JS_SRC})
 set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "" SUFFIX ".node")
 target_link_libraries(${PROJECT_NAME} PRIVATE ${CMAKE_JS_LIB} kuzu)
+
+# Include Node-API wrappers
+execute_process(COMMAND node -p "require('node-addon-api').include"
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        OUTPUT_VARIABLE NODE_ADDON_API_DIR
+        )
+string(REPLACE "\n" "" NODE_ADDON_API_DIR ${NODE_ADDON_API_DIR})
```

Comment:
Please do not try to minify the scripts in this way. If we need to do this, probably we should use webpack instead of doing this manually.